JavaScriptSPA/authConfig.js

```diff
@@ -0,0 +1,16 @@
+const msalConfig = {
+    auth: {
+        clientId: "e760cab2-b9a1-4c0d-86fb-ff7084abd902", //This is your client ID
+        authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/" + b2cPolicies.signUpSignIn, //This is your tenant info
+        validateAuthority: false
+    },
+    cache: {
+        cacheLocation: "localStorage",
+        storeAuthStateInCookie: true
+    }
+};
+
+// request to sign-in (returns an idToken)
+const loginRequest = {
```

Comment:
Why is request here if the file is named authConfig.js?

Reply:
same as above. move to auth.js instead?